models/data_warehouse_l.model.lkml

```diff
@@ -119,13 +69,12 @@ explore: user_events_telemetry {
   }
 
   join: license_server_fact {
+    view_label: "License Server Fact"
     relationship: many_to_one
-    sql_on: CASE WHEN ${user_events_telemetry._dbt_source_relation2}
-    IN ('"ANALYTICS".EVENTS.CLOUD_PORTAL_PAGEVIEW_EVENTS', '"ANALYTICS".EVENTS.PORTAL_EVENTS')
-    THEN COALESCE(${user_events_telemetry.portal_customer_id},${user_events_telemetry.context_traits_portal_customer_id})
-    = ${license_server_fact.customer_id}
-    ELSE ${user_events_telemetry.user_id} = ${license_server_fact.server_id} END
-    ;;
+    sql_on: COALESCE(${user_events_telemetry.portal_customer_id},${user_events_telemetry.context_traits_portal_customer_id})
+          = ${license_server_fact.customer_id}
+          LEFT JOIN blp.license_server_fact as license_server_fact2 ON
+          ${user_events_telemetry.userid} = license_server_fact2.server_id ;;
```

Comment:
this join will lead to rows fanning out since server_id is not a primary key in license_server_fact.